### PR TITLE
fakedeath

### PR DESCRIPTION
**Pull request by adacovsk**
Fakedeath a little less fake-looking
:cl:
 * tweak: improved fakedeath used in Changeling's regenerative stasis and zombiepowder

### CODE REVIEW

**Comment by SonixApache** [comment]
i don't know what the fuck `fakedeath` is, i doubt the average player does - your :cl: is not descriptive

**Comment by gurfan** [comment]
oxyloss will never be more than 200

**Comment by SonixApache** [comment]
better but still not super descriptive but whatever

File: code/game/objects/items/devices/scanners.dm [hunk 1]
```diff
@@ -169,13 +169,12 @@ Subject's pulse: ??? BPM"})
...
 		user.visible_message("<span class='notice'>[user] analyzes [M]'s vitals.</span>", ignore_self = TRUE)
 		playsound(user, 'sound/items/healthanalyzer.ogg', 50, 1)
-	var/fake_oxy = max(rand(1, 40), M.getOxyLoss(), (300 - (M.getToxLoss() + M.getFireLoss() + M.getBruteLoss())))
 	var/OX = M.getOxyLoss() > 50   ? "<b>[M.getOxyLoss()]</b>"   : M.getOxyLoss()
 	var/TX = M.getToxLoss() > 50   ? "<b>[M.getToxLoss()]</b>"   : M.getToxLoss()
 	var/BU = M.getFireLoss() > 50  ? "<b>[M.getFireLoss()]</b>"  : M.getFireLoss()
 	var/BR = M.getBruteLoss() > 50 ? "<b>[M.getBruteLoss()]</b>" : M.getBruteLoss()
 	if(M.status_flags & FAKEDEATH)
-		OX = fake_oxy > 50 ? "<b>[fake_oxy]</b>" : fake_oxy
+		OX = "<b>[200]</b>"
```
Comment (Kurfursten) [inline code/game/objects/items/devices/scanners.dm]: remove []

**Pull request merged**